cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -315,7 +315,7 @@ func ValidateMixedArguments(flag *pflag.FlagSet) error {
 
 	mixedInvalidFlags := []string{}
 	flag.Visit(func(f *pflag.Flag) {
-		if f.Name == "config" || strings.HasPrefix(f.Name, "ignore-checks-") || strings.HasPrefix(f.Name, "skip-") || f.Name == "dry-run" || f.Name == "kubeconfig" {
+		if f.Name == "config" || strings.HasPrefix(f.Name, "ignore-preflight-") || strings.HasPrefix(f.Name, "skip-") || f.Name == "dry-run" || f.Name == "kubeconfig" {
```

Comment:
Good catch!
one nit: since we have only `ignore-preflight-errors` with prefix `ignore-preflight-`, this can be changed to `f.Name == "ignore-preflight-errors"`. @dims WDYT?
It's OK as-is anyway. (We may have other flags with such prefix in the future?)

Reply:
will turn that into a full string check @xiangpengzhao